legal-api/tests/unit/models/test_review.py

```diff
@@ -45,7 +45,7 @@ def test_review_save(session):
     review.filing_id = filing.id
     review.nr_number = filing_dict['filing']['continuationIn']['nameRequest']['nrNumber']
     review.identifier = filing_dict['filing']['continuationIn']['foreignJurisdiction']['identifier']
-    review.completing_party = 'completing party'
+    review.contact_details = 'contact@email.com'
```

Comment:
By the way, email.com is an actual, working domain.

I think it's better to use example.com for sample emails as that's a non-working domain.

Reply:
`never.get` is also acceptable. Thanks for the change.